_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-mailchimp.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Mailchimp: Get Mailchimp Status.
+ * API to determine if current site has linked Mailchimp account and mailing list selected.
+ * This API is meant to be used in Jetpack and on WPCOM.
+ *
+ * @since 7.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Mailchimp extends WP_REST_Controller {
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'mailchimp';
+		$this->wpcom_is_wpcom_only_endpoint = true;
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			array(
+				array(
+					'methods'  => WP_REST_Server::READABLE,
+					'callback' => array( $this, 'get_mailchimp_status' ),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Check if MailChimp is set up properly.
+	 *
+	 * @return bool
+	 */
+	private function is_connected() {
+		$option = get_option( 'jetpack_mailchimp' );
+		if ( ! $option ) {
+			return false;
+		}
+		$data = json_decode( $option, true );
+		if( ! $data ) {
```

Comment:
Nitpick: you are missing a space before the opening ( here.

Reply:
Fixed in 187029c95ab6add2d8f8ffad39faa0c27817027c